components/collective-page/sections/Budget.js

```diff
@@ -36,6 +36,14 @@ const budgetSectionAccountFieldsFragment = gqlV2/* GraphQL */ `
         valueInCents
         currency
       }
+      consolidatedBalance {
+        valueInCents
+        currency
+      }
+      balanceWithBlockedFunds {
+        valueInCents
+        currency
+      }
```

Comment:
I'm a bit worried about the impact of fetching these 2 balances (which, from my understanding are not cached?) for the collective page with large accounts (mainly, `opencollective`). But we can eventually optimize afterward if it gets too bad.